codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -185,6 +195,22 @@ public final T build(Http2Connection connection) {
                 reader = new Http2InboundFrameLogger(reader, frameLogger);
                 writer = new Http2OutboundFrameLogger(writer, frameLogger);
             }
+
+            // Configure the local flow controller.
+            if (connection.local().flowController() == null) {
+                connection.local()
+                        .flowController(new DefaultHttp2LocalFlowController(connection,
+                                DEFAULT_WINDOW_UPDATE_RATIO, autoRefillConnectionWindow));
```

Comment:
Actually having this in the builder is kinda awkward since the concept currently only exists for just the default implementation of `Http2LocalFlowController`. Should there be a setter on `Http2LocalFlowController` to control `autoRefillConnectionWindow`?